yarn.lock

```diff
@@ -2571,6 +2571,11 @@ chrome-trace-event@^1.0.0:
   dependencies:
     tslib "^1.9.0"
 
+ci-info@^1.5.0:
```

Comment:
wut is this?

Reply:
Oh, weird. Looks like our existing yarn.lock had a missing dependency.

```
jest-cli@^25.1.0:
  dependencies:
    is-ci "^2.0.0"

jest-util@^25.1.0:
  dependencies:
    is-ci "^2.0.0"

cypress@^3.1.4:
  dependencies:
    is-ci "1.2.1"

is-ci@1.2.1:
  dependencies:
    ci-info "^1.5.0"

is-ci@^2.0.0:
  dependencies:
    ci-info "^2.0.0"

ci-info@^2.0.0
```

It was missing `ci-info "^1.5.0"`